profiler/src/main/java/com/splunk/opentelemetry/profiler/ThreadDumpToStacks.java

```diff
@@ -16,84 +16,50 @@
 
 package com.splunk.opentelemetry.profiler;
 
-import java.util.Spliterator;
-import java.util.Spliterators;
-import java.util.function.Consumer;
-import java.util.stream.Stream;
-import java.util.stream.StreamSupport;
-
 /**
  * This class turns the "wall of stacks" from the jdk.ThreadDump event into Stream<String>, where
```

Comment:
This comment is no longer true. How about renaming the class to `NextRegionFinder` or sth similar?

Reply:
Or, maybe the `findNext()` method could be an instance method of `ThreadDumpRegion`? There would be no need to expose its fields then